discovery-provider/alembic/versions/6cf96b71cf3d_add_playlist_last_added_to.py

```diff
@@ -0,0 +1,24 @@
+"""add-playlist-last-added-to
+
+Revision ID: 6cf96b71cf3d
+Revises: d9992d2d598c
+Create Date: 2021-05-14 01:30:20.006813
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '6cf96b71cf3d'
+down_revision = 'd9992d2d598c'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('playlists', sa.Column('last_added_to', sa.DateTime(), nullable=True))
```

Comment:
for consistency, maybe last_added_at?

Reply:
Would there be any point in backfilling the column and make it non-nullable? 
I think the only benefit is that you don't have to check for the field being type None when comparing dates below.
I guess you wouldn't know the date so it would have to be set to something like created_at which isn't great or do something like comparing the previous `playlist_contents` column of previous entries which seems like a large query.
I think not backfilling the column fine - just wanted to entertain the idea.

Reply:
@raymondjacobson I think there is a small nuance with `at` vs `to`. last_added_at would seem to suggest the addition of the playlist, while last_added_to suggest an addition to the playlist, as in "a track was last added to" this playlist. I'm fine with either tho

Reply:
yeah to is good w/ me